src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_VALUE = 40;
+  const BASIC_RENT = RENT_VALUE * days;
```

Comment:
this var is not the constant 'cause the value depends on the days argument

so let name it in camelCase